Setting/Factions/CADMUS (Collective for the Advancement of Directed Morphogenetic Synthesis).md

```diff
@@ -0,0 +1,43 @@
+---
+tier: "5"
+allies:
+  - "[[Intership Council]]"
+  - "[[Navigation]]"
+  - Ship Defense Corps
+  - Scrappers
+enemies:
+  - "[[The Coven]]"
+demonym: cadmus
+aliases:
+  - CADMUS
+---
+#faction
+
+## Background
+Once known as the [[Foundation For Municipal Health]], this organization was borne from the need to supply medical care to generations of people. Over population becoming a concern the organization shifted focus from conventional medicine and extending human life, and focused on more eldritch research. There are extra test subjects afterall.. On the discovery of a synthetic mutagenic fungus, there was extensive research in its use in augmenting human capabilities and its side effects. Some seek immortality so they may experience what awaits in orbit of the [[Final Star]], while others examine unlocking the full potential of the human brain and cybernetics
+
+[[The Coven]] opposes their silo. Their knowledge belongs to the people, and bears wider use and experimentation. They extensively use what scraps of secrets they have uncovered, primarily from the sole few who have 'defected' to their cause.
+
+## Turf
+The location of all parts of [[The Academy]] are not common knowledge, hidden from the public to avoid fear of what their work comprises and to reduce reprisal. These include residences, although some choose to reside in the [[Garden District]]
+
+## NPCs
+[[Dr. Xavier Baleford]] - Working on the research for metabolic optimization, if people need to eat and breath less, perhaps the ship can hold out a little longer.
+
+[[Dr. Angeline Angleheart]] - Working on a cure for [[Psychogenic Encephalopathy]], however the lab is about to leak, what escapes into the population?
+
+[[Dr. Aaron Mudd]] Someone who is working on naturally grown manipulators and augmentations, who needs an extra hand when you have three!
+## Notable Assets
+High security laboratory for studying hazardous biologicals, and an industrial euthanasia & reclamation system for test subjects..
+## Situation
+Over population grips the ship, people are dying from lack of resources and at the current rate the ship will have insufficient resources to reach the [[Final Star]]. Research efforts may complete soon enough to change course, but good work takes time, and cases of [[Psychogenic Encephalopathy]] are on the rise. 
+
+## Clocks
+
+`clock 3 / 10` Lab is barely contained, a leak is imminent
```

Comment:
None of these clock values are extremely final, I wasn't totally sure what numbers I want here